qa/distros/all/rhel_9.0.yaml

```diff
@@ -0,0 +1,7 @@
+os_type: rhel
```

Comment:
Is there a need to still have the rhel yamls with this change?

Reply:
No, but I felt it was harmless to include them for the future. I can remove the commit if you like.

Reply:
Let's add these yamls where we have rhel9 fog images.

Reply:
done